icmplib/sockets.py

```diff
@@ -74,9 +79,10 @@ class ICMPSocket:
     _ICMP_ECHO_REQUEST       = -1
     _ICMP_ECHO_REPLY         = -1
 
-    def __init__(self, address=None, privileged=True):
+    def __init__(self, address=None, privileged=True, interface=None):
```

Comment:
Nit: you can force keyword arguments by inserting an asterisk.

```suggestion
    def __init__(self, address=None, *, privileged=True, interface=None):
```

It makes it easier to read code by preventing someone to write `ICMPSocket(address, False)`, instead forcing `ICMPSocket(address, privilegied=False)`.

Reply:
Ah, cool! I didn't know about that function. 
I also prefer it when the keyword is included; it's annoying to have to check what it stands for or be forced to set it.

Since the whole library doesn't use it, would it be unclean to only use it here? And it would be a breaking change.

Reply:
Yes it's a breaking change so should start with internal functions and only do it for newly added keywords.